### PR TITLE
feat: implement wizard-like UI for the Image Factory

### LINKED ISSUE #111: Image Factory: wizard-like UI
Refactor the Web UI to make it more wizard-like.

### PR DESCRIPTION
**Pull request by smira**
Break down the UI into the series of steps, be smart about available choices for each variant, provide links to the documentation.

Fixes #111

### CODE REVIEW

**Comment by smira** [comment]
[Screencast from 2024-05-14 21-06-50.webm](https://github.com/siderolabs/image-factory/assets/55211/0b8ed2aa-4a21-482c-a690-c29026fd4ae1)

File: internal/frontend/http/templates/wizard-final.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
+{{ define "wizard-final" }}
+{{ template "hidden-inputs" . }}
+{{ template "header" "Schematic Ready" }}
```
Comment (Unix4ever) [inline internal/frontend/http/templates/wizard-final.html:3]: Maybe? But let's see what our native speakers say :)

```suggestion
{{ template "header" "Schematic is Ready" }}
```

**Comment by smira** [comment]
> It looks like the video got corrupted at 1:10

yeah, looks like it got corrupted on my end :disappointed:

File: internal/frontend/http/templates/wizard-final.html (hunk 1 above)
Comment (smira) [reply internal/frontend/http/templates/wizard-final.html:3]: :man_shrugging:

**Review by frezbo — APPROVED** [review]

**Comment by smira** [comment]
Fixed screncast:

[Screencast from 2024-05-15 18-08-27.webm](https://github.com/siderolabs/image-factory/assets/55211/933a3cac-3fe6-4836-a6c4-360c8e71a91c)

**Pull request merged**